tests/test_2327_array_interface.py

```diff
@@ -13,7 +13,8 @@ def test_wrap_index_cupy():
     data = cp.arange(10, dtype=cp.int64)
     index = ak.index.Index64(data)
     other_data = cp.asarray(index)
-    assert cp.shares_memory(data, other_data)
+    result = cp.shares_memory(data, other_data)
+    assert result is True
```

Comment:
We shouldn't have tests in the `tests` directory that depend on CuPy. This should be in the `tests-cuda` directory.

There seem to be two of these:

```console
% fgrep -r 'importorskip("cupy' tests
tests/test_2649_dlpack_support.py:    cp = pytest.importorskip("cupy")
tests/test_2327_array_interface.py:    cp = pytest.importorskip("cupy")
```

If the whole file is about CuPy tests, the whole file can just be moved to the `tests-cuda` directory; if it's just one test, it can be taken out of that file and put into a close-enough file in `tests-cuda` or a new one—it's okay for a file to have only one test. (Also, the `cp = importorskip("cupy")` should be converted to a top-of-the-module `import cupy as cp`.)

Reply:
I agree, the tests are in the wrong place.

Reply:
though, it makes sense to have the `wrap <numpy, cupy, and list>` tests in one place.

Reply:
done